source/connection.txt

```diff
@@ -6,14 +6,9 @@ Connection Guide
 
 .. toctree::
 
-   Connect to MongoDB </connection/connect>
-   Connection Options </connection/connection-options>
-   MongoClient Settings </connection/mongoclientsettings>
-   Stable API </connection/stable-api>
-   Network Compression </connection/network-compression>
-   JNDI Datasource </connection/jndi>
+   Create a MongoClient </connection/connect>
+   Specify Connection Options </connection/specify-connection-options>
    Connection Troubleshooting </connection/connection-troubleshooting>
-   AWS Lambda <https://www.mongodb.com/docs/atlas/manage-connections-aws-lambda/>
 
 .. contents:: On this page
    :local:
```

Comment:
S: Can't comment on the lines directly, but I think the bullet list of pages needs to be updated: it can just include Create a MongoClient, Specify Connection Options, and Connection Troubleshooting